cmd/oras/internal/display/handler.go

```diff
@@ -113,7 +113,7 @@ func NewPullHandler(out io.Writer, format option.Format, path string, tty *os.Fi
 func NewDiscoverHandler(out io.Writer, format option.Format, path string, rawReference string, desc ocispec.Descriptor, verbose bool) (metadata.DiscoverHandler, error) {
 	var handler metadata.DiscoverHandler
 	switch format.Type {
-	case option.FormatTypeTree.Name, "":
```

Comment:
I think we can remove `""` since it's just to be defensive on a non-existent code path.
```suggestion
	case option.FormatTypeTree.Name:
```